openpype/modules/default_modules/ftrack/ftrack_module.py

```diff
@@ -357,3 +356,14 @@ def tray_exit(self):
     def set_credentials_to_env(self, username, api_key):
         os.environ["FTRACK_API_USER"] = username or ""
         os.environ["FTRACK_API_KEY"] = api_key or ""
+
+    def get_task_time(self, project_name, asset_name, task_name):
+        session = self.create_ftrack_session()
+        query = (
+            'Task where name is "{}"'
+            ' and parent.name is "{}"'
+            ' and project.full_name is "{}"'
+        ).format(task_name, asset_name, project_name)
+        task_entity = session.query(query).one()
```

Comment:
You should not expect that you will find the task.
```suggestion
        task_entity = session.query(query).first()
        if not task_entity:
            return 0
```